newsroom/templates/base_layout.html

```diff
@@ -172,4 +178,18 @@ <h3 class="a11y-only">{{ gettext('Side Navigation') }}</h3>
 {% endblock %}
 
 </body>
+<script>
```

Comment:
this code should go probably rather to assets/index.ts and it should use `gettext` for those titles